src/mokernels/slfm.jl

```diff
@@ -0,0 +1,60 @@
+@doc raw"""
+    LatentFactorMOKernel(
+        g::AbstractVector{<:Kernel}, 
+        e::AbstractVector{<:Kernel}, 
+        A::AbstractMatrix
+        )
+
+A semiparametric kernel for problems involving multiple response variables.
+
+``k((x, p), (y, p)) = k_p(x, y) = \Sum^{Q}_{q=1} A_{pq}g_q(x, y) + e_p(x, y)``
+
+# Arguments
+- `g::AbstractVector{<:Kernel}`: an array of kernels
```

Comment:
Would help to make this a little more descriptive:
```suggestion
- `g::AbstractVector{<:Kernel}`: a vector of kernels, one for each latent process
```